docs/learning-sessions.md

```diff
@@ -0,0 +1,10 @@
+##  Learning Sessions
+
+- For the most up to date information, please visit the [Sig Security Meeting Notes](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg).
+- Also, please check out the slack channel at [#otel-sig-security](https://cloud-native.slack.com/archives/C05A85QC281).
+
+Here is the list of learning sessions hosted by the sig-security community:
+
+| **Date**   | **Topic**                                                              | **Speaker(s)**                                                                                          | **Link**                                     |
+| ---------- | :--------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------------- | :------------------------------------------: |
+| TBD        | [cosign](https://github.com/sigstore/cosign)                           | [Carlos Tadeu Panato Junior](https://github.com/cpanato)                                                |                                              |
```

Comment:
@cpanato, do you remember when was the presentation you've given to the Collector SIG? Would you be open to doing another one for the SIG Security? We'd then record and make it available for other folks in the community.

Reply:
I don't remember exactly what i spoke but we can redo that :)

Reply:
Thanks @cpanato! would be great to have you at next Wednesday's SIG call (29-Nov) if that works for you!

Reply:
@codeboten sorry for this week and next I cannot join :(, but I can plan to join in two weeks